tests/support/helpers.py

```diff
@@ -1675,7 +1675,8 @@ def run(self, *args, **kwargs):
         kwargs.setdefault("universal_newlines", True)
         env = kwargs.pop("env", None)
         if env:
-            env = self.environ.copy().update(env)
+            env = self.environ.copy()
+            env.update(env)
```

Comment:
This variant of my fix for the `.update` issue introduces another one since you're reassigning the provided `env` kwarg with the copy of the env var dict and update the latter with itself. `USE_STATIC_REQUIREMENTS` is thus still absent, which is why the tests are still failing.

You could either get rid of the branching since it's relatively unnecessary (like I did) or rename the first `env` from kwargs to something else.

Reply:
Odd. I thought I updated this to do it in 2 statements you show here.

Reply:
Oh nevermind I see what you are saying. I made the change too quickly I guess without really looking at it.

Reply:
Thanks for your help.